ios/Classes/Model/Characteristic.m

```diff
@@ -1,5 +1,5 @@
 #import "Characteristic.h"
-#import "CharacteristicResponse.h"
+#import "CharacteristicResponse1.h"
```

Comment:
BlemulatorCharacteristicResponse?

Reply:
Changed according to the review suggestion.